ext/opentelemetry-ext-http-requests/src/opentelemetry/ext/http_requests/__init__.py

```diff
@@ -103,12 +98,12 @@ def instrumented_request(self, method, url, *args, **kwargs):
 
             span.set_attribute("http.status_code", result.status_code)
             span.set_attribute("http.status_text", result.reason)
+            span.set_status(
+                Status(_http_status_to_canonical_code(result.status_code))
+            )
 
             return result
 
-        # TODO: How to handle exceptions? Should we create events for them? Set
```

Comment:
I think this TODO is still relevant?

Reply:
I think this comment was added at the very beginning when the spans were not able to set the status based on an exception.